Assets/Scripts/UI/StatsUI.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Services;
+using TMPro;
+using UnityEngine;
+
+public class StatsUI : MonoBehaviour
+{
+
+    private EasyService<ScoreManager> _scoreManager;
+
+    public TextMeshProUGUI PawnPiecesDestroyed;
+    public TextMeshProUGUI RookPiecesDestroyed;
+    public TextMeshProUGUI BishopPiecesDestroyed;
+    public TextMeshProUGUI KnightPiecesDestroyed;
+    public TextMeshProUGUI KingPiecesDestroyed;
+    public TextMeshProUGUI QueenPiecesDestroyed;
+    public TextMeshProUGUI CurrencyText;
+    public TextMeshProUGUI TimePlayedText;
+    public TextMeshProUGUI TurnsLastedText;
+    public TextMeshProUGUI UnitsLostText;
+    
```

Comment:
Only Advice I'd have is to also store references to the Images in the UI.

And then assigning the images on run time from a data object. so that when we chang eout sprites, we only have to do it in one location and all the spites across the game will be updated.

The sprites can be found at  `GlobalGameAssets.Instance.BishopData.Sprite;`

just swop out `BishopData `for whichever piece you need.